translate/storage/mozilla_lang.py

```diff
@@ -116,6 +130,13 @@ def parse(self, lines):
     def serialize(self, out):
         if self.is_active or self.mark_active:
             out.write(b"## active ##\n")
+        for header in self._headers:
+            out.write(bytes("%s\n" % header))
+        if self._headers:
+            out.write(b"\n\n")
         for unit in self.units:
             out.write(six.text_type(unit).encode('utf-8'))
             out.write(b"\n\n\n")
+
+    def getlangheaders(self):
```

Comment:
I wonder if instead of `self._headers` it is better to just have `self.headers`. That way we can skip this extra method.